zsh-aws-vault.plugin.zsh

```diff
@@ -16,44 +16,61 @@ alias avs='aws-vault server'
 # Convenience Functions                                              #
 #--------------------------------------------------------------------#
 function avl() {
-  aws-vault login $@
+  local aws_profile=${1:-$AWS_VAULT}
+  aws-vault login $aws_profile
 }
 
 function avll() {
-  aws-vault login -s $@
+  local aws_profile=${1:-$AWS_VAULT}
+  aws-vault login -s $aws_profile
+}
+
+function aven() {
+  local aws_profile=${1:-$AWS_VAULT}
+  unset AWS_ACCESS_KEY_ID
+  unset AWS_DEFAULT_REGION
+  unset AWS_REGION
+  unset AWS_SECRET_ACCESS_KEY
+  unset AWS_SECURITY_TOKEN
+  unset AWS_SESSION_EXPIRATION
+  unset AWS_SESSION_TOKEN
+  unset AWS_VAULT
+  export $(aws-vault exec $aws_profile -- env | grep AWS)
 }
 
 function ave() {
   aws-vault exec $@
 }
 
 function avsh() {
+  local aws_profile=${1:-$AWS_VAULT}
```

Comment:
Does this fallback make sense in this function? I think if you're already logged into a profile, you'll receive an error when you try to run this again.

```
 17:26:04  ~  avsh default
 17:28:47  ☁  ~  avsh
aws-vault: error: exec: aws-vault sessions should be nested with care, unset $AWS_VAULT to force
```